### PR TITLE
Lyche 'under construction' front page

### LINKED ISSUE #508: Replace "Direktesending" with some link to Lyche
I propose that we replace the "Direktesending" menu item with some link that goes to Lyche. [UKA](https://uka.no) uses `Mat` and I think that would be a great menu name. One could also consider `Lyche`, but that might not be familiar to everybody. A third option might be `Reservation`, but it doesn't really communicate that it's about Lyche or food.

We should do this because "Direktesending" is rarely visited, and one has other means of making sure that people are able to go to the live show, for example by using a front page hijack.

### PR DESCRIPTION
**Pull request by evgiz**
Fixes #508 

Initial front page for Lyche 'pusser opp/under construction'

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -172,6 +172,7 @@
     end
 
     namespace :sulten, path: "lyche" do
+      get "/" => "lyche#pusser_opp"
```
Comment (eirikvaa) [inline config/routes.rb]: Could you translate this to English (as well as the other places where 'pusser opp' is used) ? ;-)

Comment (evgiz) [reply config/routes.rb]: Done

File: app/controllers/sulten/lyche_controller.rb [hunk 2]
```diff
@@ -1,10 +1,8 @@
-# frozen_string_literal: true
```
Comment (eirikvaa) [inline app/controllers/sulten/lyche_controller.rb]: Travis is going to complain if you don't add this back.

**Review by eirikvaa — APPROVED** [review]
LGTM

**Pull request merged**